doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -322,10 +325,9 @@ There are multiple ways to generate and register device certificates:
    .. tab:: nRF70: Script generated private key
 
       .. warning::
-         When generating the private key on your computer, make sure to keep it secure and not to share it with anyone.
-         If the private key is compromised, the security of the device is compromised.
```

Comment:
Please keep the sentence about compromised keys. I've asked security about their input, including this warning.